### DISCUSSION TITLE
Bug: 单点登录，本地回调不好用

### DISCUSSION THREAD

**Discussion by lizhangwt** [vbenjs/vue-vben-admin#4527] [General]
### Version

Vben Admin V5

### Describe the bug?

我在web-antd文件夹下add了public，新增了login.html作为单点登录的回调地址，但是本地运行的时候根本没走这个html，
打包之后login.html和index.html同级别，所以调用到login.html，我怀疑是不在同一目录级别导致的本地调用不到的问题；
但是把index.html放到public（和login同级别）后，项目起来，页面一直转圈，没看到有要加载出的迹象；
我又把login.html放到web-antd的子目录下，项目起来了但是页面不好用
求求各位大神，单点登录，怎么能保证各个环境的保持一直呀

### Reproduction

node20

### System Info

```shell
pnpm
```

### Validations

- [X] Read the [docs](https://anncwb.github.io/vue-vben-admin-doc/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by anncwb** [comment]
不是项目的问题，你应该去查阅现有的vite如何实现

**Reply by elarbor** [reply]
这个项目的后端项目在哪啊？
